### ISSUE TITLE
Handle CERFA for tax reduction

### ISSUE THREAD

**Issue by MartinDelille** [liberapay/liberapay.com#383]
It'll be nice to handle automatic CERFA generation for project that can benefit it. Of course this is totally specific to France, but it could be a starting point.

Here is the CERFA document: http://www.impots.gouv.fr/portal/deploiement/p1/fichedescriptiveformulaire_5185/fichedescriptiveformulaire_5185.pdf

We'll need real users identity (not pseudo).

**Comment by Changaco** [comment]
Tagging as Blocked by #271.

**Comment by MartinDelille** [comment]
I don't think it is related to revealing the identity. I think the real identity should be an option that could be keep hidden and used only for the CERFA generation.

**Comment by Changaco** [comment]
Delivering the receipt to the donor requires revealing their identity to the nonprofit that received the money. #271 is about breaking anonymity in a different way though, so I guess this is a related but separate issue.

**Comment by MartinDelille** [comment]
Yes indeed!

**Comment by ArchangeGabriel** [comment]
For instance Framasoft is a “public utility” non-profit organization in France, so you can have a refund on your taxes up to 66 % of your donation to them. But not through Liberapay, which is a blocker for me to use this service when it comes to giving to such organizations.

**Comment by Changaco** [comment]
Relevant blog post from Donorbox: [How to Create a 501(c)(3) Tax-Compliant Donation Receipt](https://donorbox.org/nonprofit-blog/create-a-501c3-tax-compliant-donation-receipt/).

**Comment by tanius** [comment]
Another relevant data point: There is a decision no. C-318/07 from 2009-01-27 by European Court of Justice (ECJ) that basically says that tax-deductible donations to charitable organizations must be possible independent of whether the recipient is inside the same member state or in a different member state of the European Union.

It seems that this is not yet really relevant in practice, as one would still have to prove that a charitable organization registered in one country would also be considered charitable according to the laws of ones own jurisdiction. But still – there is at least in principle a way to have tax-deductible donations going to other EU member states.

**Comment by Changaco** [comment]
French nonprofits which provide the kind of receipts this issue is about are now obligated to declare them to the state.

*[Associations : obligation de déclaration des dons et reçus avant le 31 décembre 2022](https://www.economie.gouv.fr/associations-obligationdeclaration-dons-recus-decembre-2022)*

A warning about that should be added to Liberapay once generating the receipts is implemented.

---

New versions of the French forms for tax deductible donations were published in 2023. The changes are described in *[Utilisez-vous les bons modèles de reçus fiscaux ? | Ministère de la Culture](https://www.culture.gouv.fr/thematiques/mecenat/articles-a-la-une/utilisez-vous-les-bons-modeles-de-recus-fiscaux)*.
